lib/src/connector/ischool_plus_connector.dart

```diff
@@ -34,6 +34,12 @@ class ISchoolPlusConnector {
   static const String _getCourseName = "${_iSchoolPlusUrl}learn/mooc_sysbar.php";
   static const _ssoLoginUrl = "${NTUTConnector.host}ssoIndex.do";
 
+  // The Authorization Step of ISchool (2023-10-21)
+  // 0. POST https://app.ntut.edu.tw/ssoIndex.do
```

Comment:
[Question]
Why this line says using "POST" method but the code actual does is using `getDataByGet` ?

Reply:
Fix on 2f24ad9